renovate.json

```diff
@@ -2,12 +2,12 @@
   "extends": [
     "config:base"
   ],
-  "schedule": ["on Monday every 2 weeks"],
+  "schedule": ["before 8am on the first day of the month"],
   "packageRules": [
     {
-      "depTypeList": [
-        "require"
-      ],
+      "depTypeList": ["require"],
+      "updateType": ["minor", "patch"],
+      "automerge": true,
```

Comment:
I'm guessing we don't want to enable automerge for this project
Plus, we'd likely run into codeowners issues anyway.

Reply:
I thought we wanted to turn on automerge, as mentioned in #227 ? 🤔

Reply:
We'd need two separate configs then, one for automerge qualifying stuff and one for now. But, absent some other kind of bot like bulldozer, I don't think we can do automerge with codeowners / branch protection enabled.
So let's get this working without any automerge enabled at the renovate level. if we want to create some labels that would let us use another tool to automerge, that could make some sense

Let's avoid using policy-bot for this use case if we can.